Provisioners/Nginx/Debian/Requirements

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exit 0
```

Comment:
to me, the call to prelude should not be implicit, because automation is much more simple if each command is run __only__ when the automating system calls it (for example, imagine we need to wrap prelude into something for specific platform)

So my idea is that Requirements needs to be sure that
1. _Prelude has been run (by checking something _Prelude exports)
2. requirements are installed (through _Prelude or through any other system)
    here, requirements for this service are
    - bats

So **each** `Requirements` file should at least test for something exported by _Prelude

And **each** `Requirements` file should test its own dependencies (even if _Prelude should have installed them)